web/src/components/netflow-table.tsx

```diff
@@ -94,12 +115,72 @@ const NetflowTable: React.FC<{
         columns={columns}
       />
       <Tbody>
-        {flows.map((f, i) => (
-          <NetflowTableRow key={i} flow={f} columns={columns} />
-        ))}
+        {error ? (
+          <ErrorState colSpan={columns.length} error={error} />
+        ) : initializing ? (
+          <LoadingContents colSpan={columns.length} />
+        ) : _.isEmpty(flows) ? (
+          <NoResultsFound colSpan={columns.length} />
+        ) : (
+          flows.map((f, i) => <NetflowTableRow key={i} flow={f} columns={columns} />)
+        )}
       </Tbody>
     </TableComposable>
   );
 };
 
+const LoadingContents: React.FC<{
+  colSpan: number;
+}> = ({ colSpan }) => {
+  return (
+    <Tr>
+      <Td colSpan={colSpan}>
+        <Bullseye>
+          <Spinner size="xl" />
+        </Bullseye>
+      </Td>
+    </Tr>
+  );
+};
+
+const NoResultsFound: React.FC<{
+  colSpan: number;
+}> = ({ colSpan }) => {
+  return (
+    <Tr>
+      <Td colSpan={colSpan}>
+        <Bullseye>
+          <EmptyState id="no-results-found" variant={EmptyStateVariant.small}>
+            <EmptyStateIcon icon={SearchIcon} />
+            <Title headingLevel="h2" size="lg">
+              No results found
```

Comment:
you need to call translation here

Check `const { t } = useTranslation('plugin__network-observability-plugin');` and call t(`No results found`) after adding your text in `plugin__network-observability-plugin.json`

Reply:
Ah yeah! I forgot it. Thanks